components/l3d/src/L3DFile.cpp

```diff
@@ -41,6 +49,15 @@
  *
  *  TODO: Investigate
  *
+ * ------------------------ start of uv2 block ---------------------------------
+ *
+ *  TODO: Investigate
```

Comment:
Uv should be vec2 format. What do you see when you print out d'une contents?

Reply:
None of the meshes we're loading currenty have uv2 data. I think it may be present in the compressed temple meshes. IMO it's worth leaving this until I tackle the next PR which is related to temples and camera paths

Reply:
Add an issue number then :)